### PR TITLE
Updated Kotlin for Data Science landing page:

### PR DESCRIPTION
**Pull request by MKhalusova**
reworked text, removed Zeppelin building from source instructions, removed instructions for KNumpy and lets-plot, removed banner image, etc.

### CODE REVIEW

File: pages/docs/reference/data-science-overview.md [hunk 1]
```diff
@@ -7,137 +7,44 @@ title: "Kotlin for Data Science"
 
 # Kotlin for Data Science
 
-![Kotlin for data science]({{ url_for('asset', path='images/landing/data-science/data-science-overview.png')}})
+From building ETL pipelines to productionizing machine learning models, Kotlin can be a great choice for
```
Comment (p7nov) [inline pages/docs/reference/data-science-overview.md:10]: Starting from specific terms may scare newcomers who just wondering about Kotlin and looking at all overview pages. 
Maybe start from some simple commonly known things?

File: pages/docs/reference/data-science-overview.md [hunk 2]
```diff
@@ -7,137 +7,44 @@ title: "Kotlin for Data Science"
 
 # Kotlin for Data Science
 
-![Kotlin for data science]({{ url_for('asset', path='images/landing/data-science/data-science-overview.png')}})
+From building ETL pipelines to productionizing machine learning models, Kotlin can be a great choice for 
+working with data:
+* Static typing and null safety help create reliable, maintainable code that is easy to troubleshoot. 
+* Kotlin is concise, readable and easy to learn.
```
Comment (p7nov) [inline pages/docs/reference/data-science-overview.md:13]: I'd put this in the first place: it's the simplest idea in the list and it's about starting the user journey (`easy to learn`)

File: pages/docs/reference/data-science-overview.md [hunk 3]
```diff
@@ -7,137 +7,44 @@ title: "Kotlin for Data Science"
 
 # Kotlin for Data Science
 
-![Kotlin for data science]({{ url_for('asset', path='images/landing/data-science/data-science-overview.png')}})
+From building ETL pipelines to productionizing machine learning models, Kotlin can be a great choice for 
+working with data:
+* Static typing and null safety help create reliable, maintainable code that is easy to troubleshoot. 
+* Kotlin is concise, readable and easy to learn. 
+* Being a JVM language Kotlin gives you great performance, and an ability to leverage an entire ecosystem
```
Comment (p7nov) [inline pages/docs/reference/data-science-overview.md:14]:
```suggestion
* Being a JVM language, Kotlin gives you great performance and an ability to leverage an entire ecosystem 
```

File: pages/docs/reference/data-science-overview.md [hunk 4]
```diff
@@ -7,137 +7,44 @@ title: "Kotlin for Data Science"
...
+working with data:
+* Static typing and null safety help create reliable, maintainable code that is easy to troubleshoot. 
+* Kotlin is concise, readable and easy to learn. 
+* Being a JVM language Kotlin gives you great performance, and an ability to leverage an entire ecosystem 
+of tried and true Java libraries. 
 
-Data science has a special place in information technology: it includes aspects of both software development and
-scientific research at the same time. As a discipline, data science covers a broad range of areas: data engineering,
-data analysis, machine learning, visualization, and many more.
+## Interactive Editors
```
Comment (p7nov) [inline pages/docs/reference/data-science-overview.md:17]:
```suggestion
## Interactive editors
```
We use sentence case in section titles

File: pages/docs/reference/data-science-overview.md [hunk 5]
```diff
@@ -7,137 +7,44 @@ title: "Kotlin for Data Science"
...
+exploratory research. Kotlin integrates with these tools to help you explore data, share your findings with 
+colleagues, or build up your data science and machine learning skills.
 
-General-purpose languages can also be applied to data science, Kotlin is already being adopted to data science.
-Here we’ll introduce you to some things that are useful to know about when using Kotlin for data science.
+### Jupyter Kotlin kernel
 
-## Tools
+The Jupyter Notebook is an open-source web application that allows you to create and share documents 
+(aka "notebooks") that can contain code, visualizations and markdown text.
```
Comment (p7nov) [inline pages/docs/reference/data-science-overview.md:26]:
```suggestion
(aka "notebooks") that can contain code, visualizations, and markdown text. 
```

File: pages/docs/reference/data-science-overview.md [hunk 6]
```diff
@@ -7,137 +7,44 @@ title: "Kotlin for Data Science"
...
-Instead, they tend to use integrated development environments (IDE) that can handle all the development tasks in a single tool.
-Data scientists have similar tools called _notebooks_. Notebooks let users conduct research and store it in a single environment.
-In a notebook, you can write narrative text next to a code block, execute the code block, and see the results in any
-format that you need: output text, tables, data visualization, and so on.
+![Kotlin in Jupyter notebook]({{ url_for('asset', path='images/landing/data-science/kotlin-jupyter-kernel.png')}})
 
-Kotlin provides integration with two popular notebooks: Jupyter and Apache Zeppelin, which both allow you to write and
-run Kotlin code blocks.
+Check out Kotlin kernel's [GitHub repo](https://github.com/Kotlin/kotlin-jupyter) for installation 
+instructions, documentation and examples.
```
Comment (p7nov) [inline pages/docs/reference/data-science-overview.md:33]:
```suggestion
instructions, documentation, and examples.
```

File: pages/docs/reference/data-science-overview.md [hunk 7]
```diff
@@ -7,137 +7,44 @@ title: "Kotlin for Data Science"
...
 
-### Jupyter kernel
+### Zeppelin Kotlin interpreter
 
-The open-source project [Jupyter](https://jupyter.org/) offers the well-known web-based development environment **Jupyter Notebook**.
-For code execution, Jupyter uses the concept of _kernels_ - components that run separately and execute the code upon request,
-for example, when you click **Run** in a notebook.
+Apache Zeppelin is a popular web-based solution for interactive data analytics. It provides strong support 
+for the Apache Spark cluster computing system, which is particularly useful for data engineering. 
+Starting with [version 0.9.0](https://zeppelin.apache.org/docs/0.9.0-preview1/) Apache Zeppelin comes with
```
Comment (p7nov) [inline pages/docs/reference/data-science-overview.md:39]:
```suggestion
Starting from [version 0.9.0](https://zeppelin.apache.org/docs/0.9.0-preview1/), Apache Zeppelin comes with 
```

**Comment by p7nov** [comment]
Please also remove `pages/docs/tutorials/zeppelin-spark-cluster.md`. It was linked from the old version of the page you update.

File: pages/docs/reference/data-science-overview.md (hunk 1 above)
Comment (MKhalusova) [reply pages/docs/reference/data-science-overview.md]: Fixed in 0589f3228c213d9072011459e408d442132c4108

File: pages/docs/reference/data-science-overview.md (hunk 2 above)
Comment (MKhalusova) [reply pages/docs/reference/data-science-overview.md]: Fixed in 0589f3228c213d9072011459e408d442132c4108

**Pull request merged**